guide/case-studies/wallet-scrutiny.md

```diff
@@ -0,0 +1,215 @@
+
+The redesign was appreciated with a lot of enthusiasm. The comments were:
+- **Clean design:** People loved the clean and neutral design. They felt that they could better relate to the content this way.
+
+- **Easy to read:** People found the methodology easy to read and mentioned that it helps to show what went wrong in the testing process.
+
+- **Details:** Developers liked the in depth explanations and the details of what went wrong.
+
+## Insights
+- **Consensus**: Design is an iterative process that requires the team being on board with the direction of the design as well as the UX strategy. Building consensus was an essential part of the collaborative process.
+
+- **UX research**: Understanding the current user experience during the redesign process helped us to understand the main user flow as well as which areas of the website created the most friction.
+
+- **Consider the users goals**: Understanding the user groups who would visit the website helped us to understand the shortest user journey that would help them to meet their main goals. This helped with the information architecture.
+
+- **Trust**: Creating trust throughout the user experience was something that we tried to incorporate using design. This was essential to keeping the user engaged and trusting the reviews.
+
+## Conclusion
+As a community we really enjoyed working with this project and sprinting towards the redesign of the website. The entire process took about six months in total and we jumped on weekly or bi-weekly calls to keep up the momentum. The redesign of a website is an intricate process that requires a team effort as well as understanding the needs and goals of the people using the website.
+
+With that said this project is not finished. While we made a lot of changes in the redesign process the next steps with this project are to get the project in front of the world, gather feedback and speak with users and continue to iterate. There is more work to be done on the expert opinions, how the project could integrate Nostr and so contributions to that effort would be much appreciated.
+
+## Acknowledgements/Gratitude
+We would like to thank the WalletScrutiny team for their willingness to collaborate as well as members of the Bitcoin Design Community, who rallied together to make the redesign happen. Christoph and Mogashni worked with the team over a longer period of time to make the redesign a reality. We’d also like to give a shout out to our anonymous copy guy. He spent a lot of time rewriting all content for length and clarity and came up with some headlines that really capture what Wallet Scrutiny does, while still being catchy and approachable.
+
+Moving forward we will continue to support the project in order to bring more awareness to wallet reproducibility in the form of education as well as support in the design.
+
+## Resources
+Previous case studies:
```

Comment:
I don't think you need to link previous case studies. They are easily accessible via the sidebar navigation and the nav links at the bottom.

Reply:
I agree. Removed with latest commit.